### PR TITLE
Dark Jesus Collab, Adding Gimmel Mobs

### PR DESCRIPTION
**Pull request by Akiyama64**
Changed Amthrax to anthrax in config spelling.

Added two new mobs for the plains biome. Also, added ghost witches to Aquila's location. They are supposed to spawn in town at night, they are slow, and they are invisible. They drop ten netherwarts that are newfriend souls when they die (the lore has no specific purpose for anything useful).

The new mobs are a dirty spider and a dark archer. THe first is a cave spider with some serious speed II, and debuffs like blindness and posion against players. They drop some good loot for how hard they are to kill. As for the dark archer, he drops sticky pistons, but he has regen III, and punch IV with fire I on the bow. The debuffs are blindness and then night vision for the player, and confusion.

The spawn rate in the desert should also buff up, with skellies remaining about the same in that area and other mobs like the solar creeper blowing more stuff up. It has seemed there aren't many mobs at all in the desert.

### CODE REVIEW

File: civcraftConfigs/gimmelconfig.yml [hunk 1]
```diff
@@ -153,16 +153,159 @@ monster:
...
+       amount: 16
+       chance: 1
+      spidereyes:
+       material: SPIDER_EYE 
+       amount: 8
+       chance: 1
+      fermspidereyes:
+       material: FERMENTED_SPIDER_EYE  
+       amount: 4
+       chance: 0.3
```
Comment (Maxopoly) [inline civcraftConfigs/gimmelconfig.yml:174]: Not sure whether drop chances for individual items are supported

File: civcraftConfigs/gimmelconfig.yml [hunk 2]
```diff
@@ -153,16 +153,159 @@ monster:
...
+      - AIR
+      - STATIONARY_LAVA
+      - STATIONARY_WATER 
+     health: 100
+     despawn_on_chunk_unload: true
+     can_pickup_items: false
+     y_spawn_range: 16
+   darkzombiearcher:
+     identifier: undeadarcher
+     type: ZOMBIE
```
Comment (Maxopoly) [inline civcraftConfigs/gimmelconfig.yml:211]: Will a Zombie actually use the bow?

File: civcraftConfigs/gimmelconfig.yml [hunk 3]
```diff
@@ -153,16 +153,159 @@ monster:
...
+         enchant: PROTECTION_ENVIRONMENTAL
+         level: 2
+      chainboots2:
+       material: CHAINMAIL_BOOTS
+       enchants:
+        Prot2:
+         enchant: PROTECTION_ENVIRONMENTAL
+         level: 2
+      skullhelm2:
+       material: SKULL
```
Comment (Maxopoly) [inline civcraftConfigs/gimmelconfig.yml]: Right now, only helmets work as helmets

File: civcraftConfigs/gimmelconfig.yml [hunk 4]
```diff
@@ -153,16 +153,159 @@ monster:
...
+       type: CONFUSION
+       level: 1
+       duration: 10s
+       chance: 1.0 
+      blinded:
+       type: BLINDNESS
+       level: 1
+       duration: 2s
+       chance: 1.0 
+     nightv:
```
Comment (Maxopoly) [inline civcraftConfigs/gimmelconfig.yml]: intendation seems wrong here

File: civcraftConfigs/gimmelconfig.yml (hunk 2 above)
Comment (darkjesusmn) [inline civcraftConfigs/gimmelconfig.yml:211]: yes

File: civcraftConfigs/gimmelconfig.yml (hunk 1 above)
Comment (darkjesusmn) [inline civcraftConfigs/gimmelconfig.yml:174]: ill test it out asap once it goes online

File: civcraftConfigs/gimmelconfig.yml (hunk 3 above)
Comment (darkjesusmn) [inline civcraftConfigs/gimmelconfig.yml]: ahh i see, in the future will it be able to use other items for a helm? i know theres a plugin called mythic mobs i use for my server and it does almost the same thing as EE but you can do soo much more to the mobs, all the ideas for these mobs can from that plugin and with that you could have mob wear just about anyhting including glass blocks on the head that is colored aswell as color the armor they wear

Comment (Maxopoly) [inline civcraftConfigs/gimmelconfig.yml]: Yeah Mythic Mobs is basically what I want the mob spawning stuff to be at some point. It's not hard to add that stuff like the glass hat and I'll eventually do it, but it's just that theres already a really long todo list with other stuff right now.

**Comment by Akiyama64** [comment]
Fixes put in. We look forward to seeing if the zombie can use a bow, and if EE supports dropchance for items.

**Pull request merged**